pkg/user/tx_client.go

```diff
@@ -434,9 +455,11 @@ func (client *TxClient) submitToSingleConnection(ctx context.Context, txBytes []
 		// Retry with updated sequence
 		retryTxBytes, err := client.resignTransactionWithNewSequence(txBytes)
 		if err != nil {
+			span.RecordError(fmt.Errorf("txclient/retryBroadcastingTx: rebroadcast error: %w", err))
 			return nil, err
 		}
 
+		span.AddEvent("txclient/retryBroadcastingTx: successfully rebroadcasted tx after sequence mismatch")
```

Comment:
[question] `retryBroadcastingTx` doesn't match a method call above. Did it mean to say `resignTransactionWithNewSequence`?